ui/static/css/common.css

```diff
@@ -1085,3 +1085,7 @@ details.entry-enclosures {
 .disabled {
     opacity: 20%;
 }
+
+audio {
+    width: 100%;
+}
```

Comment:
The convention seems to be to always include a trailing newline (the project could use an `.editorconfig` file to help)

Reply:
ok